pylint/lint/__init__.py

```diff
@@ -43,8 +43,6 @@
 # Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
@Pierre-Sassoulas We should keep these two lines in each file since they are not added by https://github.com/PCManticore/copyrite (which is the tool I was using for adding the copyright notices).

Reply:
OK, I'm going to fix this, sorry.